test_functionality/range.test.py

```diff
@@ -0,0 +1,24 @@
+import unittest
+from functionality.range import Range
+
+
+class TestRange(unittest.TestCase):
```

Comment:
Very nice documentation of the functionality via tests :)
Each stage is evident from the tests!

Reply:
Thanks sir